core/src/test/java/org/elasticsearch/search/suggest/SuggestBuilderTests.java

```diff
@@ -39,13 +42,44 @@
 import org.elasticsearch.search.suggest.phrase.SmoothingModel;
 import org.elasticsearch.search.suggest.phrase.StupidBackoff;
 import org.elasticsearch.search.suggest.term.TermSuggestionBuilder;
+import org.elasticsearch.search.suggest.term.TermSuggestionBuilderTests;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
 
 import java.io.IOException;
 import java.util.Collections;
+import java.util.HashMap;
+import java.util.Locale;
 import java.util.Map.Entry;
 
+import static org.elasticsearch.common.xcontent.XContentFactory.jsonBuilder;
+import static org.hamcrest.Matchers.equalTo;
+
 public class SuggestBuilderTests extends WritableTestCase<SuggestBuilder> {
 
+    private static NamedWriteableRegistry namedWriteableRegistry;
+    private static IndicesQueriesRegistry queriesRegistry;
+    private static ParseFieldMatcher parseFieldMatcher;
+    private static Suggesters suggesters;
+
+    /**
+     * Setup for the whole base test class.
+     */
+    @BeforeClass
+    public static void init() throws IOException {
+        namedWriteableRegistry = new NamedWriteableRegistry();
```

Comment:
Nit: you could move the stuff from provideNamedWritableRegistry() here and then in provideNamedWritableRegistry() return the internal NamedWritableRegistry. Maybe less error prone when maintaining the test later to only have one registry around.